test/parallel/test-os.js

```diff
@@ -116,8 +116,8 @@ const interfaces = os.networkInterfaces();
 switch (platform) {
   case 'linux':
   {
-    const filter = (e) => e.address === '127.0.0.1';
-    const actual = interfaces.lo.filter(filter);
+    const fltr = (e) => e.address === '127.0.0.1' && e.netmask === '255.0.0.0';
```

Comment:
Can you leave the name as `filter` and just break the line on the `&&` operator.